src/forcing.jl

```diff
@@ -0,0 +1,28 @@
+struct Forcing{Tu,Tv,Tw,TT,TS}
+  u::Tu
+  v::Tv
+  w::Tw
+  T::TT
+  S::TS
+end
+
+@inline zero_func(u, v, w, T, S, Nx, Ny, Nz, Δx, Δy, Δz, i, j, k) = 0
```

Comment:
Is it slower to write `@inline zero_func(args...) = 0`? This is less explicit, but does not require maintenance.

Reply:
Done.